index.js

```diff
@@ -8,6 +8,7 @@ const { CleanWebpackPlugin } = require('clean-webpack-plugin')
 const WorkboxPlugin = require('workbox-webpack-plugin')
 const defaultCache = require('./cache')
 const { exit } = require('process')
+const log = require('./log')
```

Comment:
Is it possible to import from `next/dist/build/output/log`? This could reduce some overhead

Reply:
> Is it possible to import from `next/dist/build/output/log`? This could reduce some overhead

`Error: Cannot find module 'next/dist/build/output/log'`